pulseapi/entries/serializers.py

```diff
@@ -50,6 +57,35 @@ class EntrySerializer(serializers.ModelSerializer):
                                          slug_field='name',
                                          queryset=Creator.objects)
 
+    bookmark_count = serializers.SerializerMethodField()
+
+    def get_bookmark_count(self, instance):
+        """
+        Get the total number of bookmarks this entry received
+        """
+        return instance.bookmarked_by.count()
+
+    is_bookmarked = serializers.SerializerMethodField()
+
+    def get_is_bookmarked(self, instance):
```

Comment:
This is the kind of magic that bothers me. I understand it, but it still feels weird.